Doberman/PipelineMonitor.py

```diff
@@ -32,19 +32,23 @@ def start_pipeline(self, name):
         if (doc := self.db.get_pipeline(name)) is None:
             self.logger.error(f'No pipeline named {name} found')
             return -1
+        if (self.db.get_pipeline_stats(name)['status']=='active'):
+            self.logger.error(f'The pipeline named {name} in already active')
+            return -1
```

Comment:
I think `return -1` will do the job, but the return value is never checked so `return` will also do perfectly well (and I would suggest removing all the other return values).

Python style is more to avoid return values like this and if something goes wrong throw an exception which can be caught anyway.